templates/iam_policy/github_sbt_dependencies_policy.json.tpl

```diff
@@ -0,0 +1,20 @@
+{
+  "Statement": [
+    {
+      "Action": [
+        "s3:PutObject",
+        "s3:ListBucket",
+        "s3:GetObject"
+      ],
+      "Effect": "Allow",
+      "Resource": [
+        "arn:aws:s3:::tdr-snapshots-mgmt/*",
+        "arn:aws:s3:::tdr-snapshots-mgmt",
+        "arn:aws:s3:::tdr-releases-mgmt/*",
+        "arn:aws:s3:::tdr-releases-mgmt"
+      ],
+      "Sid": ""
+    }
+  ],
+  "Version": "2012-10-17"
```

Comment:
Maybe move the `Version` field to beginning of policy as per the other policy declarations?